packages/x-components/src/x-modules/extra-params/components/snippet-config-extra-params.vue

```diff
@@ -65,7 +73,7 @@
       currency,
       ...snippetExtraParams
     }: SnippetConfig): void {
-      forEach(snippetExtraParams, (name, value) => {
+      forEach({ ...snippetExtraParams, ...this.values }, (name, value) => {
         if (this.notAllowedExtraParams.includes(name)) {
```

Comment:
Just noticed that we are using both destructuring and an array named `notAllowedExtraParams` for discarding parameters, I think we should pick one of those options. We can do it in another PR if you prefer so. 
Also, `scope` parameter is discarded, and its solely purpose is to be sent to the API to group requests depending on the device. We can do it in another PR if you wish so.